frontend/packages/dev-console/src/components/health-checks/ProbeForm.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { FormikProps, FormikValues, useFormikContext } from 'formik';
+import { ButtonVariant, TextInputTypes } from '@patternfly/react-core';
+import { CheckIcon, CloseIcon } from '@patternfly/react-icons';
+import { InputField, DropdownField, FormFooter } from '@console/shared';
+import { HTTPProbeTypeForm, TCPProbeTypeForm, CommandProbeTypeForm } from './ProbeTypeForms';
+import { ProbeType } from './health-checks-types';
+import FormSection from '../import/section/FormSection';
+import './ProbeForm.scss';
+
+const getProbeTypeForm = (value: string, ports: {}) => {
+  switch (value) {
+    case ProbeType['HTTP GET']:
+      return <HTTPProbeTypeForm ports={ports} />;
+    case ProbeType['Container Command']:
+      return <CommandProbeTypeForm />;
+    case ProbeType['TCP Socket']:
+      return <TCPProbeTypeForm ports={ports} />;
+    default:
+      return undefined;
+  }
+};
+
+export interface ProbeFormProps {
+  containerPorts?: {};
+}
+
+const ProbeForm: React.FC<FormikProps<FormikValues> & ProbeFormProps> = ({
+  handleSubmit,
+  handleReset,
+  status,
+  isSubmitting,
+  errors,
+  containerPorts,
+}) => {
+  const { values, setFieldValue } = useFormikContext<FormikValues>();
+  const onProbeTypeChange = (value) => {
+    setFieldValue('probeType', value);
+  };
+  return (
+    <div className="odc-heath-check__probe-form">
+      <FormSection>
+        <InputField
+          type={TextInputTypes.number}
+          name="failureThreshold"
+          label="Failure Threshold"
+          style={{ maxWidth: '100%' }}
+        />
+        <DropdownField
+          name="probeType"
+          label="Type"
+          items={ProbeType}
+          title={ProbeType['HTTP GET']}
+          onChange={onProbeTypeChange}
+          fullWidth
+        />
+        {getProbeTypeForm(values.probeType, containerPorts)}
+        <InputField
+          type={TextInputTypes.number}
+          name="initialDelaySeconds"
+          label="Initial Delay"
+          helpText="How long to wait before the container starts before checking it's health."
+          inputGroupText="seconds"
+          style={{ maxWidth: '100%' }}
+        />
+        <InputField
+          type={TextInputTypes.number}
+          name="periodSeconds"
+          label="Period"
+          helpText="How often to perform the probe."
+          inputGroupText="seconds"
+          style={{ maxWidth: '100%' }}
+        />
+        <InputField
+          type={TextInputTypes.number}
+          name="timeoutSeconds"
+          label="Timeout"
+          helpText="How long to wait for the probe to finish, if the time is exceeded, the probe is considered failed."
+          inputGroupText="seconds"
+          style={{ maxWidth: '100%' }}
+        />
+        <InputField
+          type={TextInputTypes.number}
+          name="successThreshold"
+          label="Success Threshold"
+          style={{ maxWidth: '100%' }}
+        />
```

Comment:
All other input fields have help text. Not for this one?

Reply:
I have worked around the High-Level Design, I would need UX's input on this one.